OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -502,7 +502,12 @@ Action InitInputPanel(Widget panel)
 					{ "ReplaySpeedSlowKey", "Slow speed" },
 					{ "ReplaySpeedRegularKey", "Regular speed" },
 					{ "ReplaySpeedFastKey", "Fast speed" },
-					{ "ReplaySpeedMaxKey", "Maximum speed" }
+					{ "ReplaySpeedMaxKey", "Maximum speed" },
+					{ "StatisticsBasicKey", "Basic Statistics" },
+					{ "StatisticsEconomyKey", "Economy Statistics" },
+					{ "StatisticsProductionKey", "Production Statistics" },
+					{ "StatisticsCombatKey", "Combat Statistics" },
+					{ "StatisticsGraphKey", "Graph Statistics" }
```

Comment:
"Graph Statistics" doesn't make sense.  Can you please change this to "Economy Graph" or "Statistics Graph"?